apps/console/src/features/applications/api/application.ts

```diff
@@ -984,3 +984,80 @@ export const getRequestPathAuthenticators = (): Promise<any> => {
                 error.config);
         });
 };
+
+/**
+ * Function to add/update My Account portal status.
+ *
+ * @param status - My Account portal status.
+ *
+ * @returns Promise of response of the My Account status update request.
+ * @throws IdentityAppsApiException
+ */
+ export const updateMyAccountStatus = (status: boolean): Promise<any> => {
```

Comment:
Let's try to avoid any. Create an interface if the return object is accessed from outside.
```suggestion
 export const updateMyAccountStatus = (status: boolean): Promise<any> => {
```